tests-schema/BasicTest.php

```diff
@@ -36,15 +25,8 @@ public function testCreateAndAlter()
             ->field('tm', ['type' => 'time'])
             ->field('dt', ['type' => 'date'])
             ->field('dttm', ['type' => 'datetime'])
-            ->field('dbl', ['type' => 'double'])
```

Comment:
don't we support double, money, enum field types anymore?

Reply:
instead of `double` use `float`, like in PHP, DBAL stores both as double (when not overrided), see https://github.com/doctrine/dbal/blob/b5e15999e8f626b9a50edef142fffc7be57d0211/lib/Doctrine/DBAL/Platforms/MySqlPlatform.php#L932